app/web/features/profile/view/UserCard.tsx

```diff
@@ -54,6 +56,7 @@ export default function UserCard({
         <TabPanel classes={{ root: classes.tabPanel }} value="about">
           <About user={user} />
         </TabPanel>
+        {modPanel}
```

Comment:
We use this component all over do we really want to already render it? Should there be some check if they are a mod just to be safe?